imports/ui/components/sidebar_menu/Sidebar.jsx

```diff
@@ -2,12 +2,13 @@ import React from 'react';
 import SidebarMenu from './SidebarMenu.jsx';
 import SidebarPanel from './SidebarPanel.jsx';
 import PanelRequirements from './PanelRequirements.jsx'
+import PanelProfile from './PanelProfile.jsx'
 
 export default class Sidebar extends React.Component {
   render() {
-    var menuPanelsList = [<SidebarPanel />, <PanelRequirements />, <SidebarPanel />, <SidebarPanel />, <SidebarPanel />];
+    var menuPanelsList = [<PanelProfile />, <PanelRequirements />, <SidebarPanel />, <SidebarPanel />, <SidebarPanel />];
```

Comment:
This is actually a great way to arrange panels in a list and render them sequentially because it's quite clear which panels will be updated!